src/models/Notifications.js

```diff
@@ -0,0 +1,16 @@
+import { Schema, model } from "mongoose";
+
+const schema = new Schema(
+  {
+    notification_id: { type: String, required: true },
+    user_id: { type: String, required: true },
+    notification_type: { type: String, required: true },
+    title: { type: String, required: true },
+    message: { type: String, required: true },
+    detail_path: { type: String, required: true },
+    created_at: { type: Date, default: Date.now },
```

Comment:
created_at is not needed in this case, because you have the timestamps set to true

Reply:
changed as suggested